### DISCUSSION TITLE
How can my app know that a user has verified their email? Can subscribing to the collection do the trick?

### DISCUSSION THREAD

**Discussion by Mashiane** [pocketbase/pocketbase#5642] [Q&A]
Hi

I'd like to have some kind of mechanism where my app can know that the user has verified their email when they do.

I am thinking of perhaps subscribing to an auth collection for this feature? Is it possible to trap such events?

Thanks

PS: What I want to do is to send emails to my users each time they verify their emails.

**Comment by dschissler** [comment]
I assume that you're speaking about Javascript [realtime API](https://pocketbase.io/docs/api-realtime/).  Subscribe to the user record and check it there.

First though you'll have to open up the [API Rules](https://pocketbase.io/docs/api-rules-and-filters/) for that collection.  After you get it working then you'll need to research how to lock it down to for example only the current user (plus admins).  Don't neglect this part or your data will be wide open.

**Comment by dschissler** [comment]
If you're talking about on the Golang server side then you'll need to make a [record event](https://pocketbase.io/docs/go-event-hooks/#db-hooks).
